### PR TITLE
Update dataset-curation.md

### PR DESCRIPTION
**Pull request by NathanMolinier**
## Description

This PR will be used to update the last changes of our data curation convention

### CODE REVIEW

File: data/dataset-curation.md [hunk 1]
```diff
@@ -551,12 +554,12 @@ sci-bordeaux
                 └── anat
                     ├── sub-001_acq-sag_T2w_label-SC_seg.nii.gz  # spinal cord (SC) binary segmentation 
                     ├── sub-001_acq-sag_T2w_label-SC_softseg.nii.gz  # spinal cord (SC) soft segmentation
-                    ├── sub-001_acq-sag_T2w_label-discs_dlabel.nii.gz  # discrete discs labeling (SC) soft segmentation
+                    ├── sub-001_acq-sag_T2w_label-discs_dlabel.nii.gz  # discrete discs labeling
```
Comment (jcohenadad) [inline data/dataset-curation.md:557]:
```suggestion
                    ├── sub-001_acq-sag_T2w_label-discs_dlabel.nii.gz  # discrete discs labeling using the following convention: https://spinalcordtoolbox.com/user_section/tutorials/vertebral-labeling/labeling-conventions.html
```

**Review by jcohenadad — CHANGES_REQUESTED** [review]
Great start! thank you @NathanMolinier. This could be added as well: https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/108#issuecomment-1892628394

**Comment by NathanMolinier** [comment]
> Great start! thank you @NathanMolinier. This could be added as well: [#108 (comment)](https://github.com/neuropoly/intranet.neuro.polymtl.ca/issues/108#issuecomment-1892628394)

Is it good like that ? [3042c39](https://github.com/neuropoly/intranet.neuro.polymtl.ca/pull/111/commits/3042c39ddb49407b3cb33a179f7865bf8bc02474)

**Review by jcohenadad — APPROVED** [review]
Amazing work ❤️

**Pull request merged**